src/pages/waypoint/api-docs/[[...page]].tsx

```diff
@@ -105,9 +105,13 @@ export async function getStaticProps({ params }) {
 		{ title: 'Waypoint', url: `/waypoint` },
 		{ title: 'API', url: `/waypoint/api-docs` },
 	]
+	// Remove the 'index' page '/waypoint/api-docs' from the breadcrumbs
+	const shouldAddBreadcrumb = !(
+		apiPageProps.operationCategory.slug === productData.slug
```

Comment:
```suggestion
		apiPageProps.operationCategory?.slug === productData.slug
```
Do I need to guard here?

Reply:
It _shouldn't_ be needed, all `operationCategory` objects should have a slug 🤔 But also can't imagine this hurting in any ways, so I'd say guard is probably better than leaving it out 👍